plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -778,6 +790,17 @@ internal class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
     animatorSet.start()
   }
 
+  override fun onSizeChanged(width: Int, height: Int) {
+    mapCenter = ScreenCoordinate((width / 2).toDouble(), (height / 2).toDouble())
+    cameraAnchor = mapCenter
+  }
+
+  override fun getAnchor() = cameraAnchor
+
+  override fun resetAnchor(anchor: ScreenCoordinate?) {
+    cameraAnchor = anchor ?: mapCenter
```

Comment:
```suggestion
    cameraAnchor = anchor ?: mapCenter
    map.jumpTo(cameraAnchor)
```

Reply:
@LukasPaczos jumping to `CameraOptions` with only `anchor` specified will actually result in nothing for user. I guess simply setting it here and performing `jumpTo` later on as part of some upcoming animation with some other camera options like zoom, bearing should work fine.

Reply:
Not relevant anymore.